PlaygroundAppUITests/XCTWaiter.Result+CustomStringConvertible.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+
+extension XCTWaiter.Result: CustomStringConvertible {
+
+    public var description: String {
+        switch self {
+        case .completed: return "XCTWaiter.Result .completed"
```

Comment:
Thanks to this extension when we `print` an `XCTWaiter.Result` value, it will print as the text defined here, as opposed to the current default `"XCTWaiter.Result"`.

I bumped into this while debugging a test failure for this work.

I decided not to make this part of the framework to avoid clashing with other `CustomStringConvertible` conformances that a user of this library might have implemented.